dist/index.html

```diff
@@ -58,11 +60,108 @@
+          <div class="px-4 py-2 m-2">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="px-4 py-2 m-2">
+            <img src="assets/img/sponsers/envato-dark.png" alt="" />
+          </div>
+          <div class="px-4 py-2 m-2">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="px-4 py-2 m-2">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="px-4 py-2 m-2">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+        </div>
+      </div>
+      <!-- Contact information -->
+
+      <div class="contact-section bg-c100 py-10">
+        <div class="container">
+          <div class="flex justify-between align-middle text-c000">
+            <!-- social icons -->
+            <div class="contact-section__social flex w-1/3">
+              <span class="hover:bg-c200 rounded-full px-3 py-2 m-1"
+                ><i class="fab fa-twitter"></i
+              ></span>
+              <span class="hover:bg-c200 rounded-full px-3 py-2 m-1"
+                ><i class="fab fa-facebook-square"></i
+              ></span>
+              <span class="hover:bg-c200 rounded-full px-3 py-2 m-1"
+                ><i class="fab fa-pinterest-p"></i
+              ></span>
+              <span class="hover:bg-c200 rounded-full px-3 py-2 m-1"
+                ><i class="fab fa-instagram"></i
+              ></span>
+            </div>
+            <!-- contact details -->
+            <div
+              class="contact-section__contact flex w-2/3 items-center justify-between text-sm"
+            >
+              <div class="contact-section__contact-mail">
+                <div class="mail flex justify-between">
+                  <div class="icon mt-1 mr-3 w-1/6">
+                    <img
+                      class="h-8"
+                      src="assets/img/contact/mail.png"
+                      alt="email logo"
+                    />
+                  </div>
+                  <div class="w-5/6">
+                    <p>needhelp@lovims.com</p>
+                    <p class="text-xxs">Email address</p>
+                  </div>
+                </div>
+              </div>
+              <div class="contact-section__contact-phone">
+                <div class="phone flex">
+                  <div class="icon mt-1 mr-3 w-1/6">
+                    <img
+                      class="h-8"
+                      src="assets/img/contact/phone.png"
+                      alt="phone logo"
+                    />
+                  </div>
+                  <div class="w-5/6">
+                    <p>666 888 6666 Phone</p>
+                    <p class="text-xxs">Line</p>
+                  </div>
+                </div>
+              </div>
+              <div class="contact-section__contact-location">
+                <div class="phone flex">
```

Comment:
I think phone class is in the wrong place here

Reply:
yes i noticed, thankes for observation